cloud/migrate-mst-cloud/entire-database.md

```diff
@@ -21,12 +21,13 @@ accordingly to avoid data loss.</highlight>
 - Create a new empty database in Timescale Cloud. For instructions on installing
   Timescale Cloud, see [Install Timescale Cloud][install-timescale-cloud].
 
+<highlight type="tip">For faster migration, [compress all your
```

Comment:
Use note, important, or warning. Tip is only available for backwards compatibility.